dq-game/src/server/controller.py

```diff
@@ -12,25 +13,26 @@ def __init__(self, socket, no_of_players):
         Creates a controller
         """
         self.socket = socket
-        self.socket.on("GAME_READY_TO_START", self.start)
         self.no_of_players = no_of_players
         self.players = []
 
-    def start(self):
-        self.trigger("START_GAME")
-
     def request_theme_choices(self, theme_list):
 
         self.socket.send_to_all(theme_list, "data-theme-list")
         self.socket.send_to_all("CHOOSE_THEME", "event")
-        self.trigger("THEMES_REQUESTED")
 
-    def get_theme_choices(self):
+    def get_theme_choices(self, theme_list):
+        self.request_theme_choices(theme_list)
         result = []
-        for message in self.socket.inbuffer:
-            if message.content_type == "THEME_CHOICE":
-                result.append(message.data)
-        print(result)
+
+        while len(result) < self.no_of_players:
+            for message in self.socket.inbuffer:
+                if message.content_type == "data-theme-choice":
+                    result.append(message.data)
+                    message.content_type = "used-data-theme-choice"
```

Comment:
Why you set another value to the content type ?

Reply:
we can remove